test/suite/test_cursor18.py

```diff
@@ -558,3 +558,54 @@ def test_skip_tombstone_on_disk(self):
         self.assertEquals(version_cursor.get_key(), 1)
         self.verify_value(version_cursor, 1, 1, WT_TS_MAX, WT_TS_MAX, 3, 0, 0, 1, 0)
         self.assertEquals(version_cursor.next(), wiredtiger.WT_NOTFOUND)
+
+    def test_multiple_keys_with_search(self):
+        self.create()
```

Comment:
Is there a way to get the old version cursor behavior - that is, just iterate through the values for a single key?  If not, that's okay - but does doc need to be updated?

Reply:
Yes, you are right. The behaviour will change. I'll update the doc if there is any.